### DISCUSSION TITLE
SECCDC Feedback

### DISCUSSION THREAD

**Discussion by hulto** [spellshift/realm#733] [General]
Thoughts and feelings about realm during SECCDC

**Comment by hulto** [comment]
Filter quests by user - Having lots of users clutters up the quests view when i'm running down a thread so being able to sort by my user or a specific train of thought would be helpful.

**Comment by tjnull** [comment]
Using the curl tome, certain binaries would upload but when the file was saved to the system, the file data was 0.

**Comment by hulto** [comment]
On first beacon hook

**Comment by hulto** [comment]
Check for reverts

**Comment by hulto** [comment]
Run a tome on every beacon everytime it callsback

**Comment by tjnull** [comment]
Having a community tome repository for those that want to contribute to Realm. It would be cool to see what other contributors have created to see them. The tavern can have an option to pull those tomes from the trusted source in case someone does not need all of them.

**Comment by hulto** [comment]
Quest names are tooo fun!
No one wants to rename them

**Comment by hulto** [comment]
searching text output is case sensitive - searching for quest names Bash_history didn't match bash_history

**Comment by hulto** [comment]
Cross platform floppy bird

**Comment by hulto** [comment]
sys.execute tome

**Comment by tjnull** [comment]
Add an in-line execute option for powershell and assembly for windows and linux

**Comment by ippsec-htb** [comment]
Better tome searching. For example, typing read should pull up the cat tomb.  Quite a few times I was trying to remember the correct terminology to download a file, or run a shell command.

**Comment by ippsec-htb** [comment]
Scheduled Quests - Be nice to have a quest not run immediately.

**Comment by ippsec-htb** [comment]
file.contains and file.replace method in eldritch would make it easier to make safer tomes.  Can easily do those functions myself, but I'm lazy if I have to do 3-4 lines of code instead of 1 to make something safe. I'll be more likely to roll the dice.

**Comment by ippsec-htb** [comment]
It would be nice to have a few pre-baked queries in the API Playground to search for common things. Like exporting the results of a quest and filtering output for something.
